### ISSUE TITLE
Stacked Chart with the wrong type of data with Google Chart API

### ISSUE THREAD

**Issue by pmay1903** [google/google-visualization-issues#2098]
I am in a trouble that the stacked chart in the left side runs well with JSFiddle. However, that code did not run with my ASP.NET. Here are the error and code. Please let me know how to address this problem.

> You called the draw() method with the wrong type of data rather than a DataTable or DataView×

[![1](http://i.stack.imgur.com/r4eft.png)](http://i.stack.imgur.com/r4eft.png)

---
## JSFIDDLE: http://jsfiddle.net/huydq91/pu5wbgpv/3/

```
<div id="div_id"></div>
```

---

```
<script type='text/javascript' src='https://www.google.com/jsapi?ext.js'>
</script><script type=text/javascript>
google.load('visualization', '1.1', {packages:['bar']});
google.setOnLoadCallback(drawChart);
function drawChart(){
var data = google.visualization.arrayToDataTable([['Month','Target Shipment','Actual Shipment'],['JAN',1053355,482899],['FEB',322087,468206],]);
var view = new google.visualization.DataView(data);
var options = {  };
var chart = new google.charts.Bar(document.getElementById('div_id'));
chart.draw(view, google.charts.Bar.convertOptions(options));
}
</script>
```

**Comment by grabks** [comment]
@pmay1903 Hi, I looked at your jsfiddle on Chrome on Mac, and I didn't see any errors. What browser/OS are you using?

**Comment by pmay1903** [comment]
@grabks Yes, it works with JSFiddle, but it did not work with my ASP.NET. I was using both FF and Chrome and my OS is Windows.

**Comment by dlaliberte** [comment]
@pmay1903 Hi, there might be an issue with the extra trailing comma in your
data, before the final ']', but if so, we are surprised by the resulting
error message and would expect a syntax error instead.  You could try to
fix that to see if it helps, however.

What you should do to figure out why your code is failing when you run it
from your asp.net server is, in your browser, view the source that is sent
to it.  You could copy the generated html and script code into jsfiddle and
the results should ideally be the same.

On Tue, Nov 3, 2015 at 7:59 PM, pmay1903 notifications@github.com wrote:

> @grabks https://github.com/grabks Yes, it works with JSFiddle, but it
> did not work with my ASP.NET. I was using both FF and Chrome and my OS is
> Windows.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/google/google-visualization-issues/issues/2098#issuecomment-153537474
> .

## 

Daniel LaLiberte https://plus.google.com/100631381223468223275?prsrc=2  -
978-394-1058
dlaliberte@Google.com dlaliberte@google.com   5CC, Cambridge MA
daniel.laliberte@GMail.com daniel.laliberte@gmail.com 9 Juniper Ridge
Road, Acton MA
